aten/src/ATen/native/mps/operations/UnaryOps.mm

```diff
@@ -201,10 +206,22 @@ static void unary_op(const Tensor& self,
     });                                                                                                          \
   }
 
+#define CREATE_MPS_FALLBACK_UNARY_TORCH_IMPL_FUNC(func_out, func_stub, func_name)                                  \
+  TORCH_IMPL_FUNC(func_out)(const Tensor& self, const Tensor& output) {                                            \
+    static const bool is_macOS_14_0_or_newer = is_macos_13_or_newer(MacOSVersion::MACOS_VER_14_0_PLUS);            \
+    if (is_macOS_14_0_or_newer) {                                                                                  \
```

Comment:
Why this check is needed? Wouldn't it be easier to just dispatch to op unconditionally on all versions of MacOS?